README.md

```diff
@@ -1,96 +1,96 @@
-# Remdev on Azure
```

Comment:
Same thing. You changed the whole File, but I don't see any changes. Did you maybe change the line endings? Please undo this.

Reply:
reverted changes caused by dos2unix